crimsobot/handlers/games.py

```diff
@@ -0,0 +1,133 @@
+from typing import List
+
+import discord
+
+from crimsobot.data.games import CRINGO_RULES, EMOJISTORY_RULES
+from crimsobot.handlers import AbstractEventGatherer, StopHandler, must_be_event
+from crimsobot.utils import cringo, tools as c
+
+
+@must_be_event('on_reaction_add')
+class CringoJoinHandler(AbstractEventGatherer):
+    def can_join_cringo(
```

Comment:
non-blocking nit: should the internal methods of these handlers should be prefixed with underscores?